saltychat/SaltyServer/VoiceManager.cs

```diff
@@ -360,6 +360,22 @@ private void OnJoinRadioChannel([FromSource] Player player, string radioChannelN
         }
         #endregion
 
+        #region Remote Events(Megaphoone)
+        [EventHandler(Event.SaltyChat_IsSpeaking)]
+        private void OnIsSpeakingMegaphone([FromSource] Player player, bool isSending)
+        {
+            if (!VoiceManager._voiceClients.TryGetValue(player, out VoiceClient voiceClient))
+                return;
+
+            string positionJson = JsonConvert.SerializeObject(voiceClient.Player.Character.Position);
+            float range = voiceClient.VoiceRange*3;
```

Comment:
I'm not sure if i like the idea of taking the voice range and multiply it by 3.
Imo it should be fixed value (like ~100), since the original idea was to use in a police vehicle.

Btw if you don't use the volume override, then there is automatic volume roll off based on the distance to the source.
This means that there should be no need for a dynamic range, since you can just whisper in the mic and will be hard to hear anyways from far away.

Reply:
The same thing, thought of a handheld device. But nice to know